rook/workflows/dd-tests.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+- id: build dd image
```

Comment:
we can make this part of getpopper/library so that we don't have to build it here, and also we expose it so other people can benefit from (or contribute to) it

Reply:
Passing the dd test commands looks cleaner by making an entrypoint script. So thats why i was building the image everytime.

Reply:
since it looks like some work, maybe we can open an issue and do it later and for all the tests.